src/components/Navbar/SettingsDrawer/PrayerTimesSection.tsx

```diff
@@ -84,13 +70,12 @@ const PrayerTimesSection = () => {
       </Section.Row>
       <Section.Row>
         <Section.Label>{t('prayer-times.location-access')}</Section.Label>
-        <RadioGroup
-          orientation={RadioGroupOrientation.Horizontal}
-          label="location access"
-          name="location-access"
-          items={locationAccessOptions}
-          value={locationAccess}
-          onChange={onLocationAccessChange}
+        <Toggle
+          isChecked={locationAccess === LocationAccess.On}
+          onClick={() => {
+            const isOn = locationAccess === LocationAccess.On;
```

Comment:
We can take `isOn = locationAccess === LocationAccess.On;` out of the callback to the parent scope since it's also used for `isChecked`